src/v/pandaproxy/schema_registry/handlers.cc

```diff
@@ -58,6 +58,9 @@ void parse_content_type_header(const server::request_t& rq) {
 
 ss::future<server::reply_t>
 get_config(server::request_t rq, server::reply_t rp) {
+    // Ensure we see latest writes
```

Comment:
In general I think these should come after the request validation (after `rq.req.reset();`)

Reply:
Done